Jenkinsfile

```diff
@@ -13,93 +13,105 @@ pipeline {
       )
   }
   stages {
-    stage('Standard build and XS2 tests') {
-      agent {
-        label 'x86_64&&macOS'
-      }
-      environment {
-        REPO = 'lib_i2s'
-        VIEW = getViewName(REPO)
-      }
-      options {
-        skipDefaultCheckout()
-      }
-      stages {
-        stage('Get view') {
-          steps {
-            xcorePrepareSandbox("${VIEW}", "${REPO}")
+    stage("Main") {
```

Comment:
Can the Jenkinsfile move to the latest version of the XMOS Jenkins Shared Library?

Reply:
I'll give it a go and see if anything breaks! On a brief overview should be fine.